.github/workflows/selfhosted.yml

```diff
@@ -29,4 +29,4 @@ jobs:
         pip install .[tiiq]
     - name: Test tii5q
       run: |
-        srun -p qpu5q pytest src/qibolab --pyargs qibolab --skip-no-qpu --platforms tii5q
+        srun -p tii5q pytest src/qibolab --pyargs qibolab --skip-no-qpu --platforms tii5q
```

Comment:
I think that this is outdated at this point given the new qpus available in the cluster.

Reply:
Now we have different qpus with 5 qubits, which one should be used for the tests?

Reply:
In main there is a different `selfhosted.yml` after we merged #235 and #218.
We don't specify anymore the platform in the yml file but the workflow is launched after putting the correct label in the pull request.
I think that you should just merge main here and fix possible conflicts.